Uninstall section refinements (#592)
elastic/docs-content: deploy-manage/uninstall.md

---
applies_to:
  deployment:
    eck: 
    ess: 
    ece: 
    self: 
---

# Uninstall

Uninstalling Elasticsearch can become necessary for several reasons. You might encounter issues during installation that you can't resolve, decommission a host, or want to start fresh with a clean installation.

Each deployment type uses a different uninstallation process. Refer to the guide that matches your setup:

> * [Uninstall Elastic Cloud Enterprise](/deploy-manage/uninstall/uninstall-elastic-cloud-enterprise.md)
> * [Uninstall Elastic Cloud on Kubernetes](/deploy-manage/uninstall/uninstall-elastic-cloud-on-kubernetes.md)

Reviewer: ```suggestion
* [Uninstall {{ece}}](/deploy-manage/uninstall/uninstall-elastic-cloud-enterprise.md)
* [Uninstall {{eck}}](/deploy-manage/uninstall/uninstall-elastic-cloud-on-kubernetes.md)
```

---
applies_to:
  deployment:
    eck: 
    ess: 
    ece: 
    self: 
  serverless:
---

# Uninstall

> Uninstalling Elastic components, such as **{{es}} clusters, deployments, or orchestrators**, may be necessary for several reasons. You might encounter installation issues that can't be resolved, need to decommission a host, or want to start fresh with a clean setup.

Reviewer 2: ```suggestion
Uninstalling Elastic components, such as {{es}} clusters, deployments, or orchestrators, may be necessary for several reasons. You might need to decommission a host, scale down a self-managed cluster, recover from an 
installation issue that can't be resolved, or start fresh with a clean setup.
```

> Uninstalling Elastic components, such as {{es}} clusters, deployments, or orchestrators, may be necessary for several reasons. You might need to decommission a host, scale down a self-managed cluster, recover from an 
> installation issue that can't be resolved, or start fresh with a clean setup.

Reviewer 2: ```suggestion
Uninstalling Elastic components, such as {{es}} clusters, deployments, or orchestrators, may be necessary for several reasons. You might need to decommission a host, scale down a self-managed cluster, recover from an installation issue that can't be resolved, or start fresh with a clean setup.
```

Different Elastic environments require different uninstallation steps. Choose the guide that matches your setup:

* Uninstall an orchestrator:

>   * [Uninstall {{ece}}](/deploy-manage/uninstall/uninstall-elastic-cloud-enterprise.md)

Reviewer 2: ```suggestion
  * [Uninstall an {{ece}} host](/deploy-manage/uninstall/uninstall-elastic-cloud-enterprise.md)
```

>   * [Uninstall {{eck}}](/deploy-manage/uninstall/uninstall-elastic-cloud-on-kubernetes.md)

Reviewer 2: ```suggestion
  * [Uninstall {{eck}} operator](/deploy-manage/uninstall/uninstall-elastic-cloud-on-kubernetes.md)
```

> * Delete a cloud deployment:
>   * [Delete a cloud deployment](/deploy-manage/uninstall/delete-a-cloud-deployment.md)
>     * [{{ech}} deployments](/deploy-manage/uninstall/delete-a-cloud-deployment.md#elastic-cloud-hosted)
>     * [Serverless projects](/deploy-manage/uninstall/delete-a-cloud-deployment.md#serverless)
>     * [{{ece}} deployments](/deploy-manage/uninstall/delete-a-cloud-deployment.md#ece)

Reviewer 2: ```suggestion
* Delete an orchestrated deployment:
  * [{{ech}} deployments](/deploy-manage/uninstall/delete-a-cloud-deployment.md#elastic-cloud-hosted)
  * [Serverless projects](/deploy-manage/uninstall/delete-a-cloud-deployment.md#serverless)
  * [{{ece}} deployments](/deploy-manage/uninstall/delete-a-cloud-deployment.md#ece)
```

:::{note}

> You can uninstall self-managed clusters, but detailed step-by-step instructions are not currently available.

Reviewer 2: ```suggestion
You can uninstall {{es}} nodes or {{kib}} instances on self-managed clusters, but step-by-step instructions are not currently available. For more details on the implications on removing {{es}} nodes, refer to [](/deploy-manage/maintenance/add-and-remove-elasticsearch-nodes.md).
```

> You can uninstall {{es}} nodes or {{kib}} instances on self-managed clusters, but step-by-step instructions are not currently available. For more details on the implications on removing {{es}} nodes, refer to [](/deploy-manage/maintenance/add-and-remove-elasticsearch-nodes.md).

Reviewer 2: ```suggestion
You can uninstall {{es}} nodes or {{kib}} instances on self-managed clusters, but step-by-step instructions are not currently available. For more details on the implications of removing {{es}} nodes, refer to [](/deploy-manage/maintenance/add-and-remove-elasticsearch-nodes.md).
```
